sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample9_RecognizeCustomEntities.cs

```diff
@@ -0,0 +1,102 @@
+            string endpoint = TestEnvironment.Endpoint;
+            string apiKey = TestEnvironment.ApiKey;
+            var client = new TextAnalyticsClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+
+            // Get input document(s).
+            string documentA = @"A recent report by the Government Accountability Office (GAO) found that the dramatic
+                                increase in oil and natural gas development on federal lands over the past six years
+                                has stretched the staff of the BLM to a point that it has been unable to meet its 
+                                environmental protection responsibilities.";
+
+            string documentB = @"David Schmidt, senior vice president--Food Safety, International Food Information 
+                                Council (IFIC), Washington, D.C., discussed the physical activity component.";
+
+            var batchDocuments = new List<TextDocumentInput>
+            {
+                new TextDocumentInput("1", documentA)
+                {
+                     Language = "en",
+                },
+                new TextDocumentInput("2", documentB)
+                {
+                     Language = "en",
+                }
+            };
+
+            //prepare actions
+            var actions = new TextAnalyticsActions()
+            {
+                RecognizeCustomEntitiesActions = new List<RecognizeCustomEntitiesAction>()
+                {
+                    new RecognizeCustomEntitiesAction(TestEnvironment.RecognizeCustomEntitesProjectName, TestEnvironment.RecognizeCustomEntitesDeploymentName)
+                }
+            };
+
+            AnalyzeActionsOperation operation = client.StartAnalyzeActions(batchDocuments, actions);
+
+            TimeSpan pollingInterval = new TimeSpan(1000);
+
+            while (!operation.HasCompleted)
+            {
+                Thread.Sleep(pollingInterval);
+                operation.UpdateStatus();
+
+                Console.WriteLine($"Status: {operation.Status}");
+                //If operation has not started, all other fields are null
+                if (operation.Status != TextAnalyticsOperationStatus.NotStarted)
+                {
+                    Console.WriteLine($"Expires On: {operation.ExpiresOn}");
+                    Console.WriteLine($"Last modified: {operation.LastModified}");
+                    if (!string.IsNullOrEmpty(operation.DisplayName))
+                        Console.WriteLine($"Display name: {operation.DisplayName}");
+                    Console.WriteLine($"Total actions: {operation.ActionsTotal}");
+                    Console.WriteLine($"  Succeeded actions: {operation.ActionsSucceeded}");
+                    Console.WriteLine($"  Failed actions: {operation.ActionsFailed}");
+                    Console.WriteLine($"  In progress actions: {operation.ActionsInProgress}");
+                }
+            }
```

Comment:
In other samples we usually poll until complete, and then print the status:
```cs
while (true)
{
    Console.WriteLine($"Status: {operation.Status}");
    operation.UpdateStatus();
    if (operation.HasCompleted)
    {
        break;
    }

    Thread.Sleep(pollingInterval);
}

// View operation status.
Console.WriteLine($"AnalyzeActions operation has completed");
Console.WriteLine();

Console.WriteLine($"Created On   : {operation.CreatedOn}");
Console.WriteLine($"Expires On   : {operation.ExpiresOn}");
Console.WriteLine($"Id           : {operation.Id}");
Console.WriteLine($"Status       : {operation.Status}");
Console.WriteLine($"Last Modified: {operation.LastModified}");
Console.WriteLine();
```

Any particular reason for doing it differently here? Same comment for the other sync sample.